Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1537,9 +1537,7 @@ def create_search_with_retry(client: Client, fetch_mode: str, offense: Dict, eve
     offense_id = offense['id']
     while num_of_failures <= max_retries:
```

Comment:
This:
`if num_of_failures == max_retries:`
(and its block) should be out of the loop.